src/Http/IsGdShortener.php

```diff
@@ -16,16 +16,16 @@ class IsGdShortener extends RemoteShortener
      * Create a new Is.gd shortener.
      *
      * @param \GuzzleHttp\ClientInterface $client
-     * @param bool $linkPreviews
+     * @param \Psr\Http\Message\UriInterface|string $baseUri
```

Comment:
Are you sure that the library should support `\Psr\Http\Message\UriInterface` as well? In my opinion `string` is enough.

Reply:
The library has done this from the start, look at the Shortener interface for instance. The reason I decided to support this is because we already use PSR responses and requests, and theres no additional setup required to make PSR URIs work with Guzzle :)